snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
ssh-keygen is not an equivalent command, sshd-keygen regenerates the host keys. A `dpkg-reconfigure openssh-server` would be closer, IIRC.